### ISSUE TITLE
Classic editor: "toggle toolbar" keyboard shortcut JS TypeError

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6198]
To reproduce:

- add a Classic block
- click the editable area
- on a mac: press `Control-Option-Z` to toggle the TinyMCE second toolbar
- `Uncaught TypeError: Cannot read property 'firstChild' of undefined`

As far as I see, this is because the WordPress script `src/wp-includes/js/tinymce/plugins/wordpress/plugin.js` expects an iframe which doesn't exist in Gutenberg.

I'm not sure the WordPress plugins script can be used "as is" in Gutenberg without a thorough check of what it does and how that behaves in Gutenberg, see for example #6195 . /Cc @azaozz @iseulde 

Aside:
why this button in Gutenberg is named "More" while traditionally it has always been named "Toolbar Toggle"? I'd be all for improving the original name but for consistency Gutenberg and the old editor should use the same name.

<img width="688" alt="screen shot 2018-04-16 at 12 00 34" src="https://user-images.githubusercontent.com/1682452/38803080-6681a114-416e-11e8-86ca-674ce5024146.png">
<img width="593" alt="screen shot 2018-04-16 at 12 00 42" src="https://user-images.githubusercontent.com/1682452/38803081-66b19d42-416e-11e8-845f-595a19d3f144.png">

**Comment by designsimply** [comment]
Tested and confirmed that pressing `Control-Option-Z` with the content area focused for a classic block doesn't toggle the more menu as expected.

> TypeError: undefined is not an object (evaluating 'b.getContentAreaContainer().firstChild')
> plugin.min.js:1:452

Additional info: I also noticed that once `Control-Option-Z` has been pressed, I can no longer click the "More" or "Toggle Toolbar" button in the classic block toolbar to trigger the 2nd row of options. ([32s](https://cloudup.com/cK5gtUgRe6z))

<sup>Tested with Safari 11.1.2 on macOS 10.13.6.</sup>

**Comment by tofumatt** [comment]
Note that `Control-Option-Z ` now removes a block and that shortcut was removed from the Classic Block:

<img width="664" alt="screenshot 2018-11-16 16 45 01" src="https://user-images.githubusercontent.com/90871/48635086-17ac6700-e9bf-11e8-9807-1a34d80d685a.png">

So it seems this is no longer an issue.

**Comment by afercia** [comment]
Yep `Uncaught TypeError: Cannot read property 'firstChild' of undefined` is still there, even if the shortcut now removes the block. Additionally, I see also a new error (testing in Chrome):
`Uncaught TypeError: Cannot read property 'getRng' of null`

guessing it's something still related to TinyMCE which tries to get the editor selection.

**Comment by afercia** [comment]
@tofumatt I'd suggest to reopen this, at least to track the issue, as releasing software that throws 2 JS TypeErrors seems not so ideal to me.

**Comment by tofumatt** [comment]
Ah, I see. @azaozz: is there a Trac issue where this is tracked or should we reopen the issue here?
